source/nanoFramework.Json/JsonConvert.cs

```diff
@@ -464,7 +531,7 @@ private static JsonArrayAttribute ParseArray(ref LexToken token)
 			}
 			var result = new JsonArrayAttribute((JsonToken[])list.ToArray(typeof(JsonToken)));
 			return result;
-		}
+		}   // end of ParseArray()
```

Comment:
No need for comments like this, as VS can easily shrink functions and this does not conform to the C# nF code style

Reply:
OK